app/views/emails/new.html.erb

```diff
@@ -1,28 +1,39 @@
 
 <h1 style='color: #E5E7E9'> Send Reminder Email</h1>
-
 <br>
-<h4 style='color: #E5E7E9'> Recipients (students who haven't filled out the form):</h1>
-<% Student.all.each do |student| %>
-   <% if student.has_participated == false %>
-      <% if student.user%>
-         <h5 style='color: #E5E7E9'> <%= student.user.email%> </h5>
-      <% end %>
-    <% end %>
-<% end %>
-
-<br>
-
 <body class="title">
 
 <%= form_for @email, html: { multipart: true } do |f| %>
```

Comment:
We can share the same form between create and edit; see how it's done for students.

Reply:
Fixed!